cmd/oras/pull.go

```diff
@@ -120,6 +120,9 @@ func runPull(opts pullOptions) error {
 			// Note: For a manifest, the 0th indexed element is always a
 			// manifest config.
 			if s.MediaType == configMediaType || (configMediaType == "" && i == 0 && isManifestMediaType(desc.MediaType)) {
+				if configPath == "" {
```

Comment:
What if a config is named but `configPath` is empty? In that case, we should download it, shouldn’t we?